Solutions/Windows Security Events/Hunting Queries/GroupAddedToPrivlegeGroup.yaml

```diff
@@ -54,4 +53,15 @@ query: |
   GroupAddition
   ) on GroupSid
   | extend timestamp = GroupCreateTime, AccountCustomEntity = Account, HostCustomEntity = Computer
```

Comment:
We need to go away from using legacy mappings so you should remove this line.

Reply:
Done